Systems/c172p-engine.xml

```diff
@@ -62,7 +62,101 @@
             </function>
             <output>/fdm/jsbsim/propulsion/engine[1]/air-intake-impedance-factor</output>
         </fcs_function>
-        
+
+    </channel>
+
+    <channel name="damage">
+        <fcs_function name="RPM over Redline">
+            <function>
+                <sum>
+                    <property>/engines/active-engine/rpm</property>
+                    <value>-2700</value>
+                </sum>
+            </function>
+            <clipto>
+                <min>0</min>
+                <max>10000</max>
+            </clipto>
+            <output>/engines/active-engine/rpm-over-redline</output>
```

Comment:
You can remove this `<output>` too. Rename the `name` attribute to something like `damage/rpm-over-redline`. Maybe there's a better prefix than `damage/`, you could look in the property tree in FG and see where the most appropriate place for the property is.